src/main/cpp/SDLPlayData.cpp

```diff
@@ -829,6 +828,12 @@ void SDLPlayData::event_loop(VideoState *is) {
 			case SDLK_a:
 				is->stream_cycle_channel(AVMEDIA_TYPE_AUDIO);
 				break;
+			case SDLK_KP_PLUS:
+				is->set_speed(1);
+				break;
+			case SDLK_KP_MINUS:
+				is->set_speed(-1);
```

Comment:
What happens when we are at the lowest value?  

When I tested at 1x speed (no speed adjustment) and press - then the clock jumps by about 100 frames forward before slowing down.  This might be caused by the way we update the clock speed.

Reply:
The clock speed is not updated at all when we are using filters, the filter will affect the presentation time (pts) of the frame before we send it to the decoder thread, Not sure why only slow motion (< 1x) rates are strangely affecting the playback (Note: might crash the player as well)